src/modules/job-info/job_state.c

```diff
@@ -35,8 +35,15 @@ struct state_transition {
     double timestamp;
 };
 
```

Comment:
Comment on the commit message.  Could we just say "job-info: get annotations from the job manager journal"?